processing/src/main/java/io/druid/query/extraction/CascadeExtractionFn.java

```diff
@@ -137,6 +142,12 @@ public ExtractionType getExtractionType()
   }
 
   @Override
+  public int arity()
+  {
+    return chainedExtractionFn.numberOfDimensionInputs();
```

Comment:
Would be more consistent to name this as `arity`, and maybe even make ChainedExtractionFn implement ExtractionFn (it doesn't right now)